draft-ietf-httpbis-digest-headers.md

```diff
@@ -337,25 +337,29 @@ The BNF for "parameter" is as is used in [RFC7230]. All digest-algorithm values
 are case-insensitive.
 
 The Internet Assigned Numbers Authority (IANA) acts as a registry for
-digest-algorithm values.  The registry contains the following tokens.
+The registry contains the tokens listed below.
 
+Some algorithms, although registered, have since been proven to be vulnerable:
+the MD5 algorithm MUST NOT be used as it has been found vulnerable to collision
+attacks [CMU-836068], the SHA algorithm is NOT RECOMMENDED as it has been found
+vulnerable to collision attacks [IACR-2019-459].
```

Comment:
how about this?
```suggestion
Some algorithms, although registered, have since been found vulnerable:
the MD5 algorithm MUST NOT be used due to collision attacks
[CMU-836068] and the SHA algorithm is NOT RECOMMENDED due to
collision attacks [IACR-2019-459].
```